include/gammu-message.h

```diff
@@ -436,6 +436,18 @@ typedef enum {
 } GSM_SMSMessageType;
 
 /**
+ * SMS message class.
+ *
+ * \ingroup SMS
+ */
+typedef enum {
+	GSM_SMS_None = -1,
+	GSM_SMS_Flash,
+	GSM_SMS_Normal,
+	GSM_SMS_USSD
```

Comment:
I'd prefer to use something out of usual range here. I know it's unlikely that there will be additional SMS class introduced, but as a safety measure, I'd choose something way off for USSD.

Reply:
@nijel Thanks. Is that what you meant?

Reply:
Yes, something like that (I was thinking about 255, but that really doesn't matter) ... probably the docs should be updated as well to match this.

Reply:
@nijel Done, thank you!

Reply:
@nijel Oops, it should be 127 because Class is unsigned char